### PR TITLE
Move files in input directory to output one without history for the moment

### PR DESCRIPTION
**Pull request by asfram**

### CODE REVIEW

File: ronal/tasks.py [hunk 1]
```diff
@@ -34,13 +34,20 @@
 
 
 def get_files(input_dir):
-    raise NotImplementedError
+    if not os.path.isdir(input_dir):
+        raise Exception('config input directory {} does not exist, cannot do anything'.format(input_dir))
 
+    files = []
+    for filename in os.listdir(input_dir):
```
Comment (patochectp) [inline ronal/tasks.py]: you can use    files = [os.path.join(input_dir, filename) for filename in os.listdir(input_dir)]

Comment (asfram) [inline ronal/tasks.py]: DONE

File: ronal/core.py [hunk 2]
```diff
@@ -44,7 +45,16 @@ def backup_files(self, files):
         """
         backup file to file system (without history for the moment)
         """
-        pass
+        output_dir = self.config.get('output_dir')
+        self.create_dir(output_dir)
+
+        for filename in files:
```
Comment (patochectp) [inline ronal/core.py]: you must not do this in the init.
files is not Initialized

Comment (datanel) [inline ronal/core.py]: @patochectp **files ** is not in **__init__** but in **backup_files** method

Comment (patochectp) [inline ronal/core.py]: you're rigth

File: ronal/core.py [hunk 3]
```diff
@@ -44,7 +45,16 @@ def backup_files(self, files):
...
+            shutil.move(filename, self.config.get('output_dir'))
```
Comment (datanel) [inline ronal/core.py]: you can use output_dir
```
shutil.move(filename, output_dir)
```

**Pull request merged**